src/tools/ip/quicip.h

```diff
@@ -0,0 +1,174 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    A client helper for resolving the public IP address.
+
+TODO:
+
+    Don't use the QUIC_ADDR format, as it's not cross-platform. Use the same
+    format as the QUIC spec's preferred address TP.
+
+--*/
+
+#include <msquichelper.h>
+
+const QUIC_API_TABLE* MsQuic;
```

Comment:
Make this extern, and then add the definition to both TU's. Otherwise including this file from multiple TUs will fail to link.

Reply:
Added to context instead.